tools/interu/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use std::{fs::OpenOptions, io::Write as _};
+
+use snafu::{report, ResultExt, Snafu};
+
+use crate::{cli::Cli, config::Config, instances::Instances};
+
+mod cli;
+mod config;
+mod instances;
+
+#[derive(Debug, Snafu)]
+enum Error {
+    #[snafu(display("failed to read config"))]
+    ReadConfig { source: config::Error },
+
+    #[snafu(display("failed to read instances file"))]
+    ReadInstances { source: instances::Error },
+
+    #[snafu(display("failed to write to output file"))]
+    WriteOutputFile { source: std::io::Error },
+}
+
+#[report]
+fn main() -> Result<(), Error> {
```

Comment:
We could use stackable-telemetry here if you wanted.

```rs
let _tracing_guard = Tracing::builder()
    .service_name("interu")
    .with_console_output(
        Settings::builder()
            .with_environment_variable("LOG_LEVEL")
            .with_default_level(tracing_subscriber::filter::LevelFilter::INFO)
            .enabled(true)
            .build()
    )
    .build()
    .init()?;
```

Reply:
Yes, that is the plan.